api/.dependency-cruiser.js

```diff
@@ -178,6 +178,25 @@ module.exports = {
       from: {},
       to: { path: "test" },
     },
+
```

Comment:
Why did we need to add this?

Reply:
Spoke with Faraz. This change was included to allow imports between certain folders in the API directory.